miri-script/src/commands.rs

```diff
@@ -479,9 +492,61 @@ impl Command {
         // Then test, and let caller control flags.
         // Only in root project as `cargo-miri` has no tests.
         e.test(".", &flags)?;
+
+        if coverage {
+            Self::show_coverage_report(&e)?;
+        }
         Ok(())
     }
 
+    fn show_coverage_report(e: &MiriEnv) -> Result<()> {
+        let profraw_files: Vec<_> = Self::profraw_files(".")?;
```

Comment:
This seems a bit messy... it will just go over all files that happened to be called `.profraw`, no matter whether they are from this run or not? Isn't there a cleaner way, like tell the instrumented binary to put the output into a particular folder?

We should not be creating files just randomly in the Miri source tree, and we certainly should not *delete* files randomly from the Miri source tree. Temporary files should go in a tempdir, and then it also shouldn't be needed to change `.gitignore`.

Reply:
I am using the `tempfile` crate now. I saw that it is used in `dev-dependencies` for some crates so maybe it is ok depending on it for the script? Otherwise I could use `std::env::temp_dir` and manually create a subdir there.

Reply:
Yeah tempfile is good. It pulls in a few more dependencies, I hope those don't have significant build time.